packages/node_modules/@webex/react-component-activity-share-files/src/index.js

```diff
@@ -9,6 +9,7 @@ import AdaptiveCard from '@webex/react-component-adaptive-card';
 import {CARD_CONTAINS_IMAGE} from '@webex/react-component-utils';
 
 import styles from './styles.css';
+import './momentum.scss';
```

Comment:
So, until we import here, the momentum overrides won't take effect. But after this, the CSS is going to stay in the `CSSOM` right?

Won't that override the CSS affecting the application where the Widget would reside? (Assuming that app also uses momentum library)